barman/config.py

```diff
@@ -961,6 +966,32 @@ def validate_server_config(self, server):
         # server section of the configuration file
         self._validate_with_keys(self._config.items(server), ServerConfig.KEYS, server)
 
+    @staticmethod
+    def _detect_missing_keys(config_items, required_keys, section):
+        """
+        Check config for any missing required keys
+
+        :param config_items: list of tuples containing provided parameters
+            along with their values
+        :param required_keys: list of required keys
+        :param section: source section (for error reporting)
+        """
+        missing_key_detected = 0
```

Comment:
It looks like the `0` and `1` values assigned to this variable are just functioning as a boolean type, in which case it'd be better to just use `False` and `True`, unless there's a particular reason for using integers which I've missed?